src/components/templates/Playa/Playa.tsx

```diff
@@ -126,10 +126,12 @@ const isPlaced = (venue: AnyVenue) => {
 
 const minZoom = () => (window.innerWidth - 2 * PLAYA_MARGIN_X) / PLAYA_WIDTH;
 
+const venues = [] as WithId<AnyVenue>[];
+
 const Playa = () => {
   // @debt This will currently load all venues in firebase into memory.. not very efficient
-  useFirestoreConnect("venues");
-  const venues = useSelector(orderedVenuesSelector);
+  // useFirestoreConnect("venues");
+  // const venues = useSelector(orderedVenuesSelector);
```

Comment:
Can we remove these, since we are 100% sure that this is a bad pattern and shouldn't be used, there is no point in having it here commented.

Reply:
This is the playa template. Not worth spending time on. It’s getting removed anyway. Commenting out is minimal effort while still vaguely explaining things